assets/cohere.yaml

```diff
@@ -145,11 +145,81 @@
       support at cohere.ai.
 
 - type: model
-  name: Cohere Representation model
+  name: Cohere Command
   # General
   organization: Cohere
   description: >
-    The Representation model is a language model trained by Cohere for
+    This model is a generative model optimized to follow commands in the prompt.
+  dependencies: [Cohere Base, coheretext]
```

Comment:
To confirm, Cohere Command uses coheretext beyond the use in constructing Cohere Base?

Reply:
Updated to make Cohere Base the only dependency here